Tests/MacOS/okta_oidc_macTests.swift

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
Should this file be there?

Reply:
Yes. It is currently empty but in future there will be OSX related unit tests